user_docs/OpenSARlab_environment.md

```diff
@@ -0,0 +1,137 @@
+# OpenSARlab Environment
+Every OpenSARlab user has access to an Amazon AWS EC2 instance. Individual instances are shared
+among groups of 1 - 3 users, depending on demand.
+
+### Operating System
+- Ubuntu 18.04
+
+### Volume (storage)
+OpenSARlab uses Amazon AWS EBS volume for user storage. After a day without use, a snapshot
```

Comment:
Some additional info about the volumes:

- Each user is allocated an EBS volume mounted to their server's home directory. 
- Volume snapshots are taken at 10:00 UTC for all volumes. 
- Only the latest snapshot is kept.
- Any volumes not used for two days are deleted. 
- If an user later starts a server and they have a volume snapshot, that snapshot will be used to create a new volume.
- After 30, 37, 41, 43 and 45 days of inactivity an email is sent out warning that
    1. the user's account will be deactivated
    2. the user's snapshot will be deleted permanently
- After 46 days of inactivity, the user's account will be deactivated and the user's snapshot will be deleted.

While 500 GB is the current volume storage allocated, smaller amounts may be allocated. Users may request a larger storage allocation on a needs basis.

Reply:
Thanks, Eric. I am adding everything on this list except for the change to volume size. We can adjust the 500GB when the change takes effect. I'll add a note next to the volume size saying "volume size subject to change." I'll also add a topic to the troubleshooting guide titled "I need my allotted storage amount increased." I'll direct users to contact an admin with a volume size increase request.